hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/pages/volumes/volumes.tsx

```diff
@@ -0,0 +1,342 @@
+    },
+    {
+      title: 'Admin',
+      dataIndex: 'admin',
+      key: 'admin',
+      sorter: (a: Volume, b: Volume) => a.admin.localeCompare(b.admin)
+    },
+    {
+      title: 'Creation Time',
+      dataIndex: 'creationTime',
+      key: 'creationTime',
+      sorter: (a: Volume, b: Volume) => a.creationTime - b.creationTime,
+      render: (creationTime: number) => {
+        return creationTime > 0 ? moment(creationTime).format('ll LTS') : 'NA';
+      }
+    },
+    {
+      title: 'Modification Time',
+      dataIndex: 'modificationTime',
+      key: 'modificationTime',
+      sorter: (a: Volume, b: Volume) => a.modificationTime - b.modificationTime,
+      render: (modificationTime: number) => {
+        return modificationTime > 0 ? moment(modificationTime).format('ll LTS') : 'NA';
+      }
+    },
+    {
+      title: 'Quota (Size)',
+      dataIndex: 'quotaInBytes',
+      key: 'quotaInBytes',
+      render: (quotaInBytes: number) => {
+        return quotaInBytes && quotaInBytes !== -1 ? byteToSize(quotaInBytes, 3) : 'NA';
+      }
+    },
+    {
+      title: 'Namespace Capacity',
+      key: 'namespaceCapacity',
+      sorter: (a: Volume, b: Volume) => a.usedNamespace - b.usedNamespace,
+      render: (text: string, record: Volume) => (
+        <QuotaBar
+          quota={record.quotaInNamespace}
+          used={record.usedNamespace}
+          quotaType='namespace'
+        />
+      )
+    },
+    {
+      title: 'Actions',
+      key: 'actions',
+      render: (_: any, record: Volume) => {
+        const searchParams = new URLSearchParams();
+        searchParams.append('volume', record.volume);
+
+        return (
+          <>
+            <Link
+              key="listBuckets"
+              to={`/Buckets?${searchParams.toString()}`}
+              style={{
+                marginRight: '16px'
+              }}>
+              Show buckets
+            </Link>
+            <a
+              key='acl'
+              onClick={() => handleAclLinkClick(record)}>
+              Show ACL
+            </a>
+          </>
+        );
+      }
+    }
+  ];
+
+  const defaultColumns = COLUMNS.map(column => ({
+    label: column.title as string,
+    value: column.key as string,
+  }));
+
+  const [state, setState] = useState<VolumesState>({
+    data: [],
+    lastUpdated: 0,
+    columnOptions: defaultColumns,
+    showPanel: false,
+    currentRow: {}
+  });
+  const [loading, setLoading] = useState<boolean>(false);
+  const [selectedColumns, setSelectedColumns] = useState<Option[]>(defaultColumns);
+  const [selectedLimit, setSelectedLimit] = useState<Option>(LIMIT_OPTIONS[0]);
+  const [searchColumn, setSearchColumn] = useState<'volume' | 'owner' | 'admin'>('volume');
+  const [searchTerm, setSearchTerm] = useState<string>('');
+
+  const loadData = () => {
+    setLoading(true);
+
+    const { request, controller } = AxiosGetHelper(
+      '/api/v1/volumes',
```

Comment:
Previously, we used to display the total count of volumes. Could you please add this feature to the new UI as well? However, let's enhance it by making it more visually appealing and including some informative text so that users clearly understand it's the total count of volumes present.

<img width="1651" alt="image" src="https://github.com/user-attachments/assets/a029bcff-ba93-403e-a50d-53ea66c1f5ac">

Reply:
I believe this was intentionally removed to keep more in line with the design language.
We already do display the number of volumes in the pagination bar - so this information becomes redundant.
Hence we had removed this from the heading as it is already present in the table pagination